editor/resources/editor/ReactContexify.css

```diff
@@ -142,42 +123,71 @@
 
 @keyframes react-contexify__flipInX {
   from {
-    transform: perspective(400px) rotate3d(1, 0, 0, 90deg);
-    animation-timing-function: ease-in;
+    transform: perspective(800px) rotate3d(1, 0, 0, 45deg);
   }
-  40% {
-    transform: perspective(400px) rotate3d(1, 0, 0, -20deg);
-    animation-timing-function: ease-in;
-  }
-  60% {
-    transform: perspective(400px) rotate3d(1, 0, 0, 10deg);
+  to {
+    transform: perspective(800px);
   }
-  80% {
-    transform: perspective(400px) rotate3d(1, 0, 0, -5deg);
+}
+@keyframes react-contexify__flipOutX {
```

Comment:
did you write all this new CSS? or rather, just updated this css file from the new version of react-contexify? I _think_ now we have better tools to include a css from node_modules, no need to copy paste the css to the editor. or maybe we only introduced css loaders for _our_ users? :)

Reply:
this css file was changed to match the utopia style, the original contexify css is not the same